script.reddit.reader/resources/ytdl_sites_porn

```diff
@@ -0,0 +1,101 @@
+3movs.com
```

Comment:
it's kind of an edge case i think.
we don't host any porn addons, as we prefer to keep our repo suitable for all ages.
but this addon is not anything like that.

perhaps just blanking this list might be a good compromise?

ping @learningit